index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
@jgoz even though the external module doesn't "convert" the declarations to ambient typings, based on your comment and on definitions generated by TypeScript compiler I tried removing the module altogether and it still works as expected.

Therefore, good idea + removed :+1:.